src/test/scala/karazin/scala/users/group/week1/homework/HomeworkSpecification.scala

```diff
@@ -47,17 +47,46 @@ object FermatNumbersSpecification extends Properties("Fermat Numbers"):
   }
 
   property("fermatNumber") = forAll { (n: Int) =>
-    fermatNumber(n) == Math.pow(2, Math.pow(2, 2)) + 1
-  }  
+    fermatNumber(2) == Math.pow(2, Math.pow(2, 2)) + 1
```

Comment:
Why don't you use generated `n`?

Reply:
Because the int type contains negative numbers, for which the Fermat numbers are not defined